XToDo/XToDoWindowController.m

```diff
@@ -265,8 +279,10 @@ - (NSView*)outlineView:(NSOutlineView*)outlineView viewForTableColumn:(NSTableCo
 
         cellView.titleField.stringValue = item.content;
         cellView.fileField.stringValue = [item.filePath lastPathComponent];
+		
+		[cellView setFixed:item.fixed];
 
-        //TODO: update 'complete' stat image
+        // TODO!: update 'complete' stat image
```

Comment:
I did think of `DONE` before, but a little modifier seems a bit nicer and quicker. Also I, personally, prefer the paradigm of only adding things, instead of altering.